src/curve_fitting/system.clj

```diff
@@ -12,8 +12,10 @@
 
 (def config
   {:state {}
-   :sketch {:state (integrant/ref :state)
-            :sketch-type :prior
+   :mode {:mode :prior}
+   :sketch {:mode (integrant/ref :mode)
+            :state (integrant/ref :state)
+            :sketch-type :resampling
```

Comment:
We might want to get rid of `:sketch-type` here and on line 32 now that we have `:mode`

Reply:
Good catch. Fortunatel, all that stuff gets removed by the next PR.